static/app/views/eventsV2/table/columnEditModal.spec.jsx

```diff
@@ -389,16 +524,14 @@ describe('EventsV2 -> ColumnEditModal', function () {
         initialData
       );
 
-      const field = newWrapper.find('QueryField input[name="refinement"]');
-      changeInputValue(field, '');
-      newWrapper.update();
-      field.simulate('blur');
+      const input = screen.getAllByRole('textbox')[2];
```

Comment:
can we check for the name of the field? maybe through an `aria-label`?

Reply:
I'll add a comment to which input it is, when I was looking I don't think it had an aria-label just a name.